renderers/HyperRendererImpl.tsx

```diff
@@ -320,15 +320,19 @@ export class HyperRendererImpl implements HyperRenderer {
 
                 const body = meta?.body;
 
+                console.log(populatedComponent.style)
+                console.log("HyperActionButton RETURN")
+
                 return (
                     <HyperActionButton
                         target={href}
                         method={method}
                         successRedirect={successRedirect}
                         failureRedirect={failureRedirect}
                         body={body}
-                        style={ populatedComponent.style ? StyleEntity.createFromDTO(populatedComponent.style).getCssStyles() : {} }
+                        css={ populatedComponent.style ? StyleEntity.createFromDTO(populatedComponent.style).getCssStyles() : {} }
```

Comment:
Seems to me that the original problem was not to pass on the `populatedComponent.style` directly...